doc/api/url.md

```diff
@@ -1469,7 +1531,14 @@ changes:
 The `url.resolve()` method resolves a target URL relative to a base URL in a
 manner similar to that of a Web browser resolving an anchor tag HREF.
 
-```js
+```mjs
+import url from 'url';
+url.resolve('/one/two/three', 'four');         // '/one/two/four'
+url.resolve('http://example.com/', '/one');    // 'http://example.com/one'
+url.resolve('http://example.com/one', '/two'); // 'http://example.com/two'
+```
+
+```cjs
```

Comment:
This describes a legacy API, I'm not sure it's worth providing an ESM example for this considering that users should avoid using those APIs.